src/utils/normalize.data.ts

```diff
@@ -0,0 +1,8 @@
+import { IUser, IUserNormalized } from '../user/user.interface';
+
+export const getNormalisedUser = (user : IUser) :IUserNormalized => {
```

Comment:
If that's the only export use `export default`

Reply:
Add documentation

Reply:
```suggestion
export const getNormalisedUser = (user: IUser): IUserNormalized => {
```

Reply:
accepted